src/style.css

```diff
@@ -129,3 +129,34 @@ h6 {
   display: inline-block;
   font-size: 1em;
 }
+
+/* FOOTER*/
+
+footer {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 80px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.social-row{
```

Comment:
jeśli chodzi o klasy to możesz opierać się na konwencji "footer__nav-link"
Dzięki temu będziemy wiedzieć że te konkretne klasy są wykorzystywane w footerze. Ułatwi to czytelność ;)

Reply:
poprawione :)